contracts/contracts/strategies/ConvexOUSDMetaStrategy.sol

```diff
@@ -0,0 +1,221 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+/**
+ * @title Curve Convex Strategy
+ * @notice Investment strategy for investing stablecoins via Curve 3Pool
+ * @author Origin Protocol Inc
+ */
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+
+import { IRewardStaking } from "./IRewardStaking.sol";
+import { IConvexDeposits } from "./IConvexDeposits.sol";
+import { ICurvePool } from "./ICurvePool.sol";
+import { IERC20 } from "./BaseCurveStrategy.sol";
+import { BaseConvexMetaStrategy } from "./BaseConvexMetaStrategy.sol";
+import { StableMath } from "../utils/StableMath.sol";
+import { IVault } from "../interfaces/IVault.sol";
+
+contract ConvexOUSDMetaStrategy is BaseConvexMetaStrategy {
+    using StableMath for uint256;
```

Comment:
We don't need this one anymore right @DanielVF ?

Reply:
I think "StableMath" is different from "SafeMath". I think we use this on for division. We look hard at it and see what it provides us.

Reply:
Oh right we need it for things like `mulTruncate`. Thx